deploy/clusteroperator-crd.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: clusteroperators.operatorstatus.openshift.io
+  name: clusterversions.operatorstatus.openshift.io
```

Comment:
clusterversions.config.openshift.io ?

Reply:
(honestly i dunno that we need to keep this file around anymore, it was a useful development tool when we stood up clusters manually but now that the installer is the only way, we can rely on the installer to have created this CRD in all our clusters)

Reply:
yeah I just as soon delete it ... will do that when I circle back to this PR (pending response to my outstanding client question I've circled back to implementing the progressing condition for image streams)